src/agent/debuglet.ts

```diff
@@ -62,6 +62,12 @@ const BREAKPOINT_ACTION_MESSAGE =
     'The only currently supported breakpoint actions' +
     ' are CAPTURE and LOG.';
 
+// PROMISE_RESOLVE_CUT_OFF_IN_SECONDS is a heuristic that we set to force the
+// debug agent return a promise when checkReady is called. The value is selected
+// between Stackdriver debugger hanging get duration (40s) and TCP default
+// time-out (https://tools.ietf.org/html/rfc5482).
+const PROMISE_RESOLVE_CUT_OFF_IN_SECONDS = 60;
```

Comment:
Nit: Make this _IN_MILLISECONDS and have its value set to 60 * 1000

Reply:
Done.